HISTORY.rst

```diff
@@ -2,6 +2,44 @@
 History
 =======
 
+3.9.0 (2020-03-16)
+------------------
+
+Critical changes:
+
+* The ``UpdateAdminProfile``/``UpdateProfile`` task has been revised to use the new Metadata ETL framework
+  as ``ProfileGrantAllAccess``. This is a breaking change for custom tasks which subclassed
+  ``UpdateAdminProfile``. The new task supports updating arbitrary Profiles in addition to
+  System Administrator.
```

Comment:
Could we add a line here about the new behavior? "Projects that require CumulusCI 3.9.0 or greater will automatically receive new behavior that updates FLS for all packaged objects, if not using a custom `package.xml` with the `update_admin_profile` tasks. Other projects may opt in with a task option; see the documentation for details."

Reply:
@davidmreed how's the updated verbiage in https://github.com/SFDO-Tooling/CumulusCI/pull/1562/commits/0feff34eff1b97e13e5d2ea087c3ff8abee4680f?